### PR TITLE
Create and test at-rule-empty-line-before; closes #155

### LINKED ISSUE #155: New rule: at-rule-empty-line-before

### PR DESCRIPTION
**Pull request by davidtheclark**

### CODE REVIEW

File: src/rules/at-rule-empty-line-before/__tests__/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  tr.ok("")
+  tr.ok("a {} b {}", "rule ignored")
+  tr.ok("@font-face {}", "first node ignored")
+  tr.ok("a {}\n\n@media {}")
+  tr.ok("@keyframes foo {}\n\n@media {}")
+
+  tr.notOk("a {} @media {}", messages.expected)
+  tr.notOk("@keyframes foo {} @media {}", messages.expected)
+  tr.notOk("a {}\n@media {}", messages.expected)
+  tr.notOk("a {}\n\n/* comment */\n@media {}", messages.expected)
```
Comment (MoOx) [inline src/rules/at-rule-empty-line-before/__tests__/index.js]: maybe this case should work ?

File: src/rules/at-rule-empty-line-before/__tests__/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+})
+
+testRule("never", tr => {
```
Comment (MoOx) [inline src/rules/at-rule-empty-line-before/__tests__/index.js]: never is kind of weird for this rules, but maybe it's just me :)

File: src/rules/at-rule-empty-line-before/__tests__/index.js (hunk 1 above)
Comment (davidtheclark) [inline src/rules/at-rule-empty-line-before/__tests__/index.js]: Is the special case you'd want (something like) always-unless-after-comment?

Comment (MoOx) [inline src/rules/at-rule-empty-line-before/__tests__/index.js]: indeed.

Comment (MoOx) [inline src/rules/at-rule-empty-line-before/__tests__/index.js]: we can do that later if people are asking for it :)

**Pull request merged**